esbuild.js

```diff
@@ -28,15 +28,24 @@ const debugAdapterRoot = path.join(
 /** @type {BuildOptions[]} */
 const configurations = [
     {
-        entryPoints: [path.join(outDir, 'extension.js')],
-        outdir: path.join(__dirname, 'dist'),
+        entryPoints: [path.join(sourceRoot, 'extension.ts')],
+        outfile: path.join(__dirname, 'dist', 'extension.cjs.js'),
         external: ['vscode'],
         format: 'cjs',
         platform: 'node',
         ...commonConfig,
     },
     {
-        entryPoints: [path.join(outDir, 'memory', 'client', 'index.js')],
+        entryPoints: [path.join(sourceRoot, 'extension.ts')],
+        outfile: path.join(__dirname, 'dist', 'extension.esm.js'),
+        external: ['vscode'],
+        plugins: [dtsPlugin()],
```

Comment:
This plugin's behavior was a little inconsistent for me locally. After pulling the branch and running the build, I got the `.d.ts` files in the `dist` directory as expected. On subsequent builds after doing `git clean -xfd && yarn && yarn build` I didn't get any `.d.ts` files. I'm looking into whether metadata is stored somewhere to try to avoid redundant builds, but any guidance on what could be going wrong there would be helpful.

Reply:
Seems to correspond to [this issue](https://github.com/Floffah/esbuild-plugin-d.ts/issues/8) in the plugin's repo.